src/imitation/scripts/common/train.py

```diff
@@ -23,6 +23,10 @@ def config():
     # Evaluation
     n_episodes_eval = 50  # Num of episodes for final mean ground truth return
 
+    # Visualization
```

Comment:
Given the config is in train, consider adding a capture function to help do the video saving?

Some of it does seem to need to be per-script (e.g. has to pass in the appropriate policy and eval environment), but checking if videos is true or not and passing in video kwargs could definitely be handled in the capture function,  might save 2-3 lines at callsites.

Reply:
True. I added `train.save_video`, which saved a few lines in train_imitation, train_adversarial, train_preference_comparisons and train_rl.